deepchecks/nlp/context.py

```diff
@@ -72,68 +73,85 @@ class _DummyModel(BasicModel):
     predictions: t.Dict[str, t.Dict[int, TTextPred]]
     proba: t.Dict[str, t.Dict[int, TTextProba]]
 
-    def __init__(self,
-                 test: TextData,
-                 y_pred_test: TTextPred,
-                 y_proba_test: TTextProba,
-                 train: t.Union[TextData, None] = None,
-                 y_pred_train: TTextPred = None,
-                 y_proba_train: TTextProba = None,
-                 model_classes: list = None,
-                 validate_data_on_predict: bool = True):
+    def __init__(
+        self,
+        *,
```

Comment:
why enable any args?

Reply:
sorry, not sure whether I get your question correctly, 
`*` is used to define keyword-only parameters ([docs](https://peps.python.org/pep-0570/#keyword-only-arguments)),
parameters that can be passed only as keywords 

IMO it will increase readibility